phc-rails/db/migrate/20150309032727_add_fields_to_person_account.rb

```diff
@@ -0,0 +1,23 @@
+class AddFieldsToPersonAccount < ActiveRecord::Migration
+  def change
+    add_column :person_accounts, :FirstName, :string
+    add_column :person_accounts, :LastName, :string
+    add_column :person_accounts, :SS_Num__c, :string
+    add_column :person_accounts, :Birthdate__c, :string
+    add_column :person_accounts, :Phone, :string
+    add_column :person_accounts, :PersonEmail, :string
+    add_column :person_accounts, :Gender__c, :string
+    add_column :person_accounts, :Identify_as_GLBT__c, :boolean
+    add_column :person_accounts, :Race__c, :string
+    add_column :person_accounts, :Primary_Language__c, :string
+    add_column :person_accounts, :Foster_Care__c, :boolean
+    add_column :person_accounts, :Veteran__c, :boolean
+    add_column :person_accounts, :Housing_Status_New__c, :string
+    add_column :person_accounts, :How_long_have_you_been_homeless__c, :string
+    add_column :person_accounts, :Where_do_you_usually_go_for_healthcare__c, :string
```

Comment:
I know this matches the salesforce columns but this might lead to unexpected behavoir in rails.  (I'm talking about the camel vs snake casing)